test/renderer-require.js

```diff
@@ -1,4 +1,17 @@
-import {rendererRequireDirect} from '../src/renderer-require';
+import {rendererRequireDirect, requireTaskPool} from '../src/renderer-require';
+
+describe('the requireTaskPool method', function() {
+  this.timeout(10*1000);
+
+  it('can make a bunch of requests at once', async function() {
+    const { getJSON } = requireTaskPool(require.resolve('../src/remote-ajax', 10));
+
+    for (let i = 0; i < 20; i++) {
+      const result = await getJSON('https://httpbin.org/get');
```

Comment:
Turns out `new Array(20)` won't be iterable (it just sets the default size). This does the trick though:

``` js
Array.apply(null, Array(20)).map(() => 0)
```